### ISSUE TITLE
Helppatches

### ISSUE THREAD

**Issue by antongit** [vvvv/VL.Stride#326]
These are still missing correct Filenames, for now these are just the short descriptions.

# Materials
- [ ] Displacement Map from the Skia's Texture
(**Blocked by:** missing NormalMap TextureFX (Maria has ported the Normal shader from beta for Node20 Materials Workshop, but we were not sure if the Algorithm there is right, so this TextureFX is not included in VL.Stride for now. Should I make a new Issue with the shader attached?)
- [ ] Layers

# Meshes
### Dynamic Meshes:

- [ ]  2 overlapping noises > Plane + Some spheres
- [ ]  2 overlapping noises > Poles

- [ ] Rutt-Etra Scan Processor with noise (https://vimeo.com/468050444)
- [ ] Rutt-Etra Scan Processor with live video from MediaFoundation
 
- [ ] Spheres (Texture Readback - CPU)
- [ ] Spheres with Instancing (Texture Readback - CPU)
- [ ] Spheres with Instancing and Shaders (GPU only) (like Stride DeepDive workshop)
- [ ] Spheres with Instancing and ShaderFX (GPU only) **(is it possible?)**

- [ ] Generative Landsape (DynamicTexture is used for Vertices and Material) (https://derivative.ca/community-post/tutorial/how-make-landscape)
(**Blocked by:** https://github.com/vvvv/VL.Stride/issues/237)

### Loading:
- [ ] Loading Mesh (OBJ - LowPoly) and setting Materials 
(**Blocked by:** https://github.com/vvvv/VL.Stride/issues/321, https://github.com/vvvv/VL.Stride/issues/307)
